### PR TITLE
Solar alignment fix

### LINKED ISSUE #35: Solar Calculator - The sun does not sit on the line
This may be iPhone X specific.

image-1-5

### PR DESCRIPTION
**Pull request by alex-taffe**
This includes changes from #38 and should not be merged until then

Fixes #35

### CODE REVIEW

File: TriggertrapSLR/SunriseSunsetCalculatorViewController.swift [hunk 1]
```diff
@@ -549,7 +549,7 @@ class SunriseSunsetCalculatorViewController: SplitLayoutViewController, CLLocati
     fileprivate func updateSunConstraint() {
         let diagramCalculations = self.whiteView.frame.size.width / 2 - self.sunriseDiagramLabel.frame.origin.x - self.sunriseDiagramLabel.frame.size.width
         let sunviewCalculations = self.sunImageView.frame.size.width / 2.0 - self.sunRotationCenterView.frame.size.width / 2.0
-        self.sunToCenterConstraint.constant =  diagramCalculations - sunviewCalculations
+        self.sunToCenterConstraint.constant =  diagramCalculations - sunviewCalculations - 10
```
Comment (Ross-Gibson) [inline TriggertrapSLR/SunriseSunsetCalculatorViewController.swift]: @alex-taffe any idea where the `10` has come from? Seems like a bit of a "Magic Number", and I'd rather avoid that if possible.

**Comment by Ross-Gibson** [comment]
@alex-taffe if you're also able to resolve the conflict would be good. Thank you.

**Comment by alex-taffe** [comment]
@Ross-Gibson the 10 came from the fact that the fonts changed (commit bf51a98ac8e38aff2a0dc2b377bb527fd5ede610). The width of the one label changed and it seemed to throw it off by exactly 10 pixels. I’ll see if I can figure out a better way to calculate the number though because I agree, it isn’t great.

**Comment by alex-taffe** [comment]
@Ross-Gibson fixed

**Pull request merged**